### PR TITLE
Add sabajyan_yerevan file with recommendations

### LINKED ISSUE #18: Create a file for sabajyan Yerevan, Armenia
- [x] Create a branch
- [x] Add commit
- [x] Open a PR
- [x] Have a discussion
- [x] Seek approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by sabajyan**
Closes #18

### CODE REVIEW

**Comment by davidjauregui** [comment]
Looks good beer academy looks like somewhere I would like to check out what street is that on?

File: sabajyan_yerevan.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Yerevan, Armenia
+
+### Places to Eat
+- Beer Academy
```
Comment (davidjauregui) [inline sabajyan_yerevan.md:4]: This is awesome

Comment (sabajyan) [reply sabajyan_yerevan.md:4]: Thank you @davidjauregui

**Comment by sabajyan** [comment]
It's on Moskovyan St., Building 8 whenever you are in Yerevan :)

**Review by allthedoll — APPROVED** [review]
🍵 is lovely.

**Pull request merged**